prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirTransformer.kt

```diff
@@ -172,15 +161,23 @@ class FhirTransformer(
      * Set the [value] on [bundleProperty] using [bundle] as the root resource and [focusResource] as the focus resource
      */
     internal fun setBundleProperty(
-        bundleProperty: String?,
+        rawBundleProperty: String?,
         value: Base,
         context: CustomContext,
         bundle: Bundle,
         focusResource: Base
     ) {
-        if (bundleProperty == null) return
+        if (rawBundleProperty == null || rawBundleProperty == "") {
```

Comment:
You can replace this with a check for .isNullOrBlank()

Reply:
Oo good call, thanks for the reminder!